client/src/pages/Home.jsx

```diff
@@ -1,10 +1,32 @@
 import React, { useEffect, useState } from 'react';
 
+import SessionSummary from '../components/SessionSummary';
+// import HomeHeader from '../components/HomeHeader';
 import { getSampleResponse } from '../utils/apiWrapper';
-
 import '../css/Home.scss';
+import '../css/SessionSummary.scss';
+import '../css/DetailsModal.scss';
```

Comment:
Remove the SessionSummary and DetailModal CSS imports. They should instead be imported in their respective component files.